Dockerfile

```diff
@@ -108,6 +110,15 @@ RUN pip install -r requirements.txt -r requirements-dev.txt
 # Jenkins runs the containers with the `-u 1000:1000` option, not as root.
 RUN chown -R +1000:+1000 /venv
 
+# spead2_net_raw is needed so that qualification tests can use ibverbs
+RUN SPEAD2_VERSION=$(grep ^spead2== requirements.txt | cut -d= -f3) && \
+    cd /tmp && \
+    wget https://github.com/ska-sa/spead2/releases/download/v$SPEAD2_VERSION/spead2-$SPEAD2_VERSION.tar.gz && \
```

Comment:
I'd keep what you have previously to just pull the file directly from Github (via the tag) rather than downloading a release (should be lighter-weight than downloading a whole tarball, 99% of which you don't need); the snippet I directed you to was intended just to show you how to set SPEAD2_RELEASE.